docs/celery.md

```diff
@@ -0,0 +1,53 @@
+
+## Background
+
+We want to upgrade Django from the current version to at least 4. But our current approach to running background tasks, namely `django-background-tasks` is no longer supported in Django 4. Hence, we decided to switch to celery for distributed tasks.
+
+
+## Developer Notes
+
+### configuration
+
+The celery configuration file is `meinberlin/config/celery.py`.
+
+Currently, we make use of the following config parameters:
+- `broker_url = "redis://localhost:6379"`
+- `result_backend = "redis"`
+- `broker_connection_retry_on_startup = True`
+- `result_extended = True`
+
+The celery app is configured from the django settings file and namespaced variables. The defaults are defined in `config/settings/base.py` but can be overriden by `config/settings/local.py`.
+
+Note that the default settings in`dev.py` enable the "always eager mode" (`CELERY_TASK_ALWAYS_EAGER = True`) in which the django server will run the shared tasks itself. This is to keep the development setup as lightweight as possible. If you want to develop or test using a celery worker make sure that you add the following to your `local.py`:
```

Comment:
can we add here --> and install and run server redis in your system, or something like this.

Reply:
sure, but this is just copied from the a+ PR, so we should probably change it there as well then

Reply:
yes, and the makefile quotes, and the readme, and the requirements base (redis goes our of a4 inherited packages)